CMakeLists.txt

```diff
@@ -132,6 +132,38 @@ if (CMAKE_BUILD_TYPE MATCHES "Coverage")
     COMMAND ${CMAKE_SOURCE_DIR}/scripts/record-coverage.sh ${CMAKE_SOURCE_DIR} ${CMAKE_BINARY_DIR})
 endif()
 
+# Enable asserts for C/C++ code if requested
+#  from the LLVM project, https://opensource.apple.com/source/llvmCore/llvmCore-2358.3/CMakeLists.txt.auto.html
```

Comment:
I'm considering moving all this into a `.cmake` file and then including it here.

Reply:
That might be better - I'm not sure.